<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1,0" columnstretch="0,0,1,0,0,0">
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>查找文件</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>位于目录</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QTableWidget" name="tableWidgetResult">
     <column>
      <property name="text">
       <string>文件名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>大小</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>修改时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>新建列</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButtonOk">
     <property name="text">
      <string>确定</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>关闭</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="pushButtonFind">
     <property name="text">
      <string>Find</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/new/icons/icons/preview1.bmp</normaloff>:/new/icons/icons/preview1.bmp</iconset>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="pushButtonBrowse">
     <property name="text">
      <string>Open</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/new/icons/icons/open3.bmp</normaloff>:/new/icons/icons/open3.bmp</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QComboBox" name="comboBoxDir"/>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QComboBox" name="comboBoxNameFilter">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>*.*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>*.cpp;*.c;*.cc</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>*.png;*.bmp;*.jpeg;*.jpg</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>FindDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>FindDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
